ecs-image-build/docker_start.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#
+# Start script for ch.gov.uk running in ecs
+
+APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+PORT=${PORT:=10000}
+
+export PERL5LIB="${APP_DIR}/local/lib/perl5"
+exec perlbrew exec perl "${APP_DIR}/script/start_app" daemon -l "http://*:${PORT}"
```

Comment:
We'll need to ensure that `perlbrew` is configured for the current shell session here:

```suggestion
source /opt/perlbrew/etc/bashrc
export PERL5LIB="${APP_DIR}/local/lib/perl5"
exec perlbrew exec perl "${APP_DIR}/script/start_app" daemon -l "http://*:${PORT}"
```